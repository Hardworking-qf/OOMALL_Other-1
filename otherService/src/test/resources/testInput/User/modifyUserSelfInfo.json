{
  "1": {
    "realName": "TEST",
    "gender": 2,
    "birthday": "2000-01-01"
  },
  "2": {
    "realName": "",
    "gender": null,
    "birthday": ""
  },
  "3": {
    "realName": "",
    "gender": "ascn9u2",
    "birthday": ""
  },
  "4": {
    "realName": "",
    "gender": null,
    "birthday": "2040-02-02"
  },
  "5": {
    "realName": "",
    "gender": null,
    "birthday": "2040-290-02"
  }
}