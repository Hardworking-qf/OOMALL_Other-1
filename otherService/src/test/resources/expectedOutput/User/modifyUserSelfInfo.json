{
  "1": {
    "response": {
      "errno": 0,
      "errmsg": "成功"
    },
    "result": {
      "errno": 0,
      "data": {
        "id": 17332,
        "userName": "testuser",
        "realName": "TEST",
        "mobile": "12300010001",
        "email": "test@test.com",
        "gender": 1,
        "birthday": "2000-01-01T00:00:00",
        "gmtCreate": "2020-11-30T09:14:39",
        "gmtModified": null
      },
      "errmsg": "成功"
    }
  },
  "2": {
    "response": {
      "errno": 0,
      "errmsg": "成功"
    },
    "result": {
      "errno": 0,
      "data": {
        "id": 17332,
        "userName": "testuser",
        "realName": "test",
        "mobile": "12300010001",
        "email": "test@test.com",
        "gender": 0,
        "birthday": null,
        "gmtCreate": "2020-11-30T09:14:39",
        "gmtModified": null
      },
      "errmsg": "成功"
    }
  }
}